### DISCUSSION TITLE
Is one large or multiple small changes via pull request preferred?

### DISCUSSION THREAD

**Discussion by MSSputnik** [spulec/moto#8191] [General]
I plan to do a couple of changes and enhancements to the QuickSight API.

From a view of the maintainer, is it better to do one large pull request to merge in all my changes or to create smaller pull request whenever a feature is implemented?

**Comment by bblommers** [comment]
Hi @MSSputnik! If all changes are related/for a single service, then a single large PR is better IMO. Easier for us to see how things are related, and easier for you as a contributor as there's no need to wait for us before you can raise followup PR's.

Edit: Good question BTW! Appreciate the fact that you ask beforehand :slightly_smiling_face:
